homeassistant/components/homekit/accessories.py

```diff
@@ -200,6 +200,10 @@ def get_accessory(  # noqa: C901
             or SensorDeviceClass.PM25 in state.entity_id
         ):
             a_type = "PM25Sensor"
+        elif device_class == SensorDeviceClass.NITROGEN_DIOXIDE:
+            a_type = "NitrogenDioxideSensor"
+        elif device_class == SensorDeviceClass.VOLATILE_ORGANIC_COMPOUNDS:
+            a_type = "VolatileOrganicCompoundsSensor"
         elif (
```

Comment:
Looks like these lines are missing test coverage. Otherwise should be good to go